src/apiMiddleware/index.js

```diff
@@ -0,0 +1,101 @@
+import {
+  DELETE_EXPRESSION,
+  LOGIN_REQUEST,
+  GET_EXPRESSIONS,
+  UPDATE_EXPRESSION_ID,
+  LOGIN_FAILURE,
+  LOGIN_SUCCESS,
+  REGISTER_REQUEST,
+  REGISTER_FAILURE,
+  REGISTER_SUCCESS,
+  FETCH_EXPRESSIONS,
+  REMOVE_EXPRESSION,
+  UPDATE_EXPRESSION,
+  LOGOUT,
+} from 'constants/actionTypes.js';
+import { query } from 'apiMiddleware/query';
+import { LOGIN_URL, EXPRESSIONS_URL, REGISTER_URL } from 'apiMiddleware/config';
+import { getJwt } from 'store/localStorage';
+
+const apiMiddleware = (store) => (next) => (action) => {
+  switch (action.type) {
+    case LOGIN_REQUEST:
+      next(action);
```

Comment:
If your action is not handled by other middlewares or reducers, you don't need to send it further (`next(action)`).